pkg/cdc/validation.go

```diff
@@ -0,0 +1,19 @@
+package cdc
+
+import (
+	cassdcapi "github.com/k8ssandra/cass-operator/apis/cassandra/v1beta1"
+)
+
+// Validate validates that the cdcConfig is valid.
+func Validate(cdcConfig *cassdcapi.CDCConfiguration) bool {
+	if cdcConfig == nil {
+		return true
+	}
+	switch {
+	case !cdcConfig.Enabled:
+		return true
+	case cdcConfig.Enabled && cdcConfig.PulsarServiceUrl == nil:
```

Comment:
Shouldn't OpenAPI validation take care of this? PulsarServiceURL minLength > 0

Reply:
Now that is a very good idea! Thanks.